streamz/core.py

```diff
@@ -304,6 +304,27 @@ def union(self, *others):
         """
         return union(children=(self,) + others)
 
+    def pluck(self, pick):
+        """ Select elements from a tuple in the stream.
+
+        Parameters
+        ----------
+        pluck : list
+            The elements to pick from the incoming element in the stream
+            (note: must be a list)
```

Comment:
This also works with non-list elements.  Perhaps `list` -> `object, list` and "The elements" -> "The element(s)"

Reply:
sounds good. I changed the "note" to:

`If an instance of list, will pick multiple elements.`